src/cargo/ops/registry.rs

```diff
@@ -245,6 +245,20 @@ pub fn publish(ws: &Workspace<'_>, opts: &PublishOpts<'_>) -> CargoResult<()> {
         }
     }
 
+    // Print success note on publish
+    if opts.registry.is_none() {
+        if let Some(ref registries) = pkg.publish() {
+            if registries[0] == CRATES_IO_REGISTRY {
```

Comment:
Checking only the first of the potential registries doesn't seem right.

We should instead check what registry we actually published to